apps/sertra/pages/index.tsx

```diff
@@ -27,7 +27,27 @@ const Home: NextPage<{ swrFallback: { [key: string]: unknown } }> = ({
         <Head>
           <title>Thunderstore Servers</title>
           <meta name="description" content="Thunderstore Modded Servers List" />
-          <link rel="icon" href="/favicon.ico" />
+          <link
+            rel="apple-touch-icon"
+            sizes="180x180"
+            href="/apple-touch-icon.png"
+          />
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="32x32"
+            href="/favicon-32x32.png"
+          />
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="16x16"
+            href="/favicon-16x16.png"
+          />
+          <link rel="manifest" href="/site.webmanifest" />
+          <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+          <meta name="msapplication-TileColor" content="#da532c" />
```

Comment:
@gkurck can you verify these color choices are intended?

Reply:
I can verify they are **not** intended. The correct value for both is `#3d70c0`.